### ISSUE TITLE
Illuminator Mk1/2 emissivity glitch when attached to Mk1 Command Pod

### ISSUE THREAD

**Issue by woodbyte** [PorktoberRevolution/ReStocked#598]
When a Mk1/2 illuminator is attached to a Mk1 Command Pod and turned on, hovering over the pod glitches the emissive map on the illuminator: https://www.twitch.tv/videos/424084842?t=03h31m03s

Hovering over the affected light temporarily fixes it, but it will be broken again if you hover over the pod.

This issue does not happen when the lights are attached to any other part.

**Comment by drewcassidy** [comment]
what other mods do you have installed? do you have Surfacelights installed?

**Comment by woodbyte** [comment]
Full list of installed mods:
![Annotation 2019-05-13 062143](https://user-images.githubusercontent.com/4104564/57597589-e1a00800-7547-11e9-9baf-6537701f4231.png)
I think the only part changing mods I'm using (besides Restock) are AntennaSleep, EngineLighting, EVAFuel and KerbalHealth, but I could be mistaken.

**Comment by ChrisAdderley** [comment]
I will look into this on the weekend, sorry I missed this.

**Comment by ChrisAdderley** [comment]
I reproduced this. I also reproduced it with every part with lights in restock, so further went back and discovered that it happens in stock too. If you have a part with ModuleLight attached to one which uses ModuleColorChanger it will have issues. If it is attached to something that uses the legacy ModuleAnimateGeneric method for cabin lights, everything seems to be ok (this includes parts like the plane cockpits).

So this is a stock bug, so I'm closing this.
